tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -873,6 +966,23 @@ function main() {
 
 		case "$action" in
 		install)
+			# Let's fail early on this, so we don't need to do a rollback
+			# in case we reach this situation.
+			if [[ -n "${EXPERIMENTAL_SETUP_SNAPSHOTTER}" ]]; then
+				if [[ "${runtime}" == "cri-o" ]]; then
+					warn "EXPERIMENTAL_SETUP_SNAPSHOTTER is being ignored!"
+					warn "Snapshotter is a containerd specific option."
+				else
+					case "${EXPERIMENTAL_SETUP_SNAPSHOTTER}" in
+						nydus)
+							;;
+						*)
```

Comment:
Ah okay - it looks like the default case is handled here early :)

Reply:
Exactly, the default case is handled early for both cases above!